packages/models-library/src/models_library/api_schemas_resource_usage_tracker/pricing_plans.py

```diff
@@ -11,13 +11,14 @@
     PricingPlanId,
     PricingUnitCostId,
     PricingUnitId,
+    UnitExtraInfo,
 )
 
 
 class PricingUnitGet(BaseModel):
     pricing_unit_id: PricingUnitId
     unit_name: str
-    unit_extra_info: dict
+    unit_extra_info: UnitExtraInfo
```

Comment:
Is this retro compatible with existing data in the database? Would those fields still validate?